mcs/restapi/sessions/sessions_test.go

```diff
@@ -0,0 +1,47 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2020 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package sessions
+
+import (
+	"testing"
+
+	mcCmd "github.com/minio/mc/cmd"
+
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
Can you add Some comments on each test similar to how we are doing for all other functions, else it is not clear what is expected or what we are trying to test. (test number, what we are testing, and the name of the function)